src/Nancy.Demo.Authentication.Forms.TestingDemo/LoginFixture.cs

```diff
@@ -17,10 +17,10 @@ public LoginFixture()
         }
 
         [Fact]
-        public void Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
+        public async Task Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
         {
             // Given, When
-            var response = browser.Post("/login/", (with) =>
+            var response = await browser.Post("/login/", (with) =>

```

Comment:
Another thought - keeping the sync methods but marked as `[Obsolete(false)]` and add the async overloads?